tests/common.py

```diff
@@ -1347,3 +1347,13 @@ def raise_contains_mocks(val):
     if isinstance(val, list):
         for dict_value in val:
             raise_contains_mocks(dict_value)
+
+
+async def create_hass_access_token(hass, user, credential):
```

Comment:
Maybe add type annotations?

Reply:
pushed in [af1fc92](https://github.com/home-assistant/core/pull/80931/commits/af1fc922c6183c751b35a758491cd34a2b7dd108)